src/bundles/csg/jscad/renderer.ts

```diff
@@ -39,13 +35,14 @@ import type {
   WrappedRendererData,
   ZoomToFitStates,
 } from './types.js';
+import { ACE_GUTTER_BACKGROUND_COLOR, ACE_GUTTER_TEXT_COLOR, BP_TEXT_COLOR } from '../../../tabs/common/css_constants.js';
 
 /* [Main] */
 let { orbit } = controls;
 
 function solidsToGeometryEntities(solids: Solid[]): GeometryEntity[] {
   let options: EntitiesFromSolidsOptions = {
-    color: hexToAlphaColor(DEFAULT_COLOR),
+    color: hexToAlphaColor('#55ffaa'),

```

Comment:
Might be good to add a comment to let people know this is the default colour, as opposed to just having this magic string around.